docs/reference/config-items.md

```diff
@@ -0,0 +1,342 @@
+| master | string | empty | Kubernetes API server address. If it is not filled in, the program will automatically determine: "empty" on the cloud, and "127.0.0.1:10550" on the edge |
+| contentType | string | empty | Type of message transmission during Kubernetes interaction. If it is not filled in, the program will automatically determine: "application/vnd.kubernetes.protobuf" on the cloud, and "application/json" on the edge |
+| qps | int32 | 100 | qps when talking with Kubernetes apiserver |
+| burst | int32 | 200 | burst when talking with Kubernetes apiserve |
+| kubeConfig | string | empty | kubeconfig file path |
+
+<a name="t1-2"></a>
+
+#### Table 1-2: commonConfig
+
+| Name | Type | Default | Description |
+| ---- | ---- | ---- | ---- |
+| mode | string | DebugMode | The operating mode (DebugMode, CloudMode, EdgeMode) that edgemesh-agent is in. Prohibit user manual configuration, automatically recognized by the program |
+| configMapName | string | edgemesh-agent-cfg | name of configmap mounted by edgemesh-agent |
+| dummyDeviceName | string | edgemesh0 | Name of the network card created by edgemesh-agent |
+| dummyDeviceIP | string | 169.254.96.16 | IP of the network card created by edgemesh-agent |
+
+<a name="t1-3"></a>
+
+#### Table 1-3: goChassisConfig
+
+| Name | Type | Default | Description |
+| ---- | ---- | ---- | ---- |
+| loadBalancer | object | [Table 1-3-1](#t1-3-1) | load balancing configuration |
+| protocol | object | [Table 1-3-2](#t1-3-2) | communication protocol configuration |
+
+<a name="t1-3-1"></a>
+
+#### Table 1-3-1: loadBalancer
+
+| Name | Type | Default | Description |
+| ---- | ---- | ---- | ---- |
+| defaultLBStrategy | string | RoundRobin | default load balancing strategy |
+| supportLBStrategies | []string | [RoundRobin, Random, ConsistentHash] | list of supported load balancing strategies  |
+| consistentHash | object | [Table 1-3-1-1](#t1-3-1-1) | consistent hash strategy |
+
+<a name="t1-3-1-1"></a>
+
+#### Table 1-3-1-1: consistentHash
+
+| Name | Type | Default | Description |
+| ---- | ---- | ---- | ---- |
+| load | float64 | 1.25 | hash ring bounded load value |
+| partitionCount | int | 100 | number of hash ring partitions |
+| replicationFactor | int | 10 | hash ring repetition factor |
+
+<a name="t1-3-2"></a>
+
+#### Table 1-3-2: protocol
+
+| Name | Type | Default | Description |
+| ---- | ---- | ---- | ---- |
+| tcpBufferSize | int | 8192 | buffer size for TCP communication |
+| tcpClientTimeout | int | 5 | TCP client timeout time, in seconds |
+| tcpReconnectTimes | int | 3 | number of retries for TCP connection establishment |
+| nodeName | string | empty | the name of the node where edgemesh-agent is scheduled. Prohibit user manual configuration, automatically recognized by the program |
+
+<a name="t1-4"></a>
+
+#### Table 1-4: modules
+
+| Name | Type | Default | Description |
+| ---- | ---- | ---- | ---- |
+| edgeDNS | object | [Table 1-4-1](#t1-4-1) | edgedns submodule |
```

Comment:
It is recommended to describe the feature of the module more detail

Reply:
done